### ISSUE TITLE
A possible bug in release 0.7

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/gpsea#339]
Just tried the new release -- see this example
See https://github.com/monarch-initiative/gpsea-cs/blob/develop/notebooks/NBAS/NBAS_gpsea.ipynb

This seems like more than a warning?
```python
 patient #1
    phenotype-features
      #0
       warnings:
       ·`time_element` is in currently unsupported format `ontology_class`
```

**Comment by ielis** [comment]
We can adjust the severity (`warn -> error`) but I guess it depends on what analysis we want to do downstream. If looking at HPO terms is all we want, then we do not need an onset. However, if we intend to analyze survival, then we need the time element.

This brings up the 2nd issue though. How do we convert an ontology class (e.g. `Neonatal onset`) into a number of days?

I think all we can do here is warn, but I am open to suggestions.

**Comment by ielis** [comment]
Fixed by #350
